package.json

```diff
@@ -148,8 +148,5 @@
     "babel-runtime": ">=6.9.0 <7.0",
     "react": "15.x.x",
     "react-dom": "15.x.x"
-  },
-  "browser": {
-    "faye": "faye/browser/faye-browser"
```

Comment:
Was this unused?

Reply:
Pretty sure it was useless, it's not even in `f/compat-expansion`.

Reply:
It might have been used in the older version of Faye, but the new one has the "browser" directive in its package.json for sure.